helm/cluster/values.schema.json

```diff
@@ -1515,6 +1515,18 @@
                                                     "type": "string",
                                                     "title": "Endpoint",
                                                     "description": "Endpoint for the container registry."
+                                                },
+                                                "insecure": {
+                                                    "type": "boolean",
+                                                    "title": "HTTP endpoint",
+                                                    "description": "Set to true to configure endpoint as HTTP instead of HTTPs (default).",
```

Comment:
```suggestion
                                                    "description": "Set to true to configure endpoint as HTTP instead of HTTPS (default).",
```

Just y nit.